fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -269,12 +274,22 @@ private boolean useOldVersionFormat() {
     /**
      * Get the provider for the record store's meta-data.
      * @return the meta-data source to use
+     * See {@link com.apple.foundationdb.record.RecordMetaDataBuilder#setLocalFileDescriptor(Descriptors.FileDescriptor)}.
```

Comment:
```suggestion
     * @see com.apple.foundationdb.record.RecordMetaDataBuilder#setLocalFileDescriptor(Descriptors.FileDescriptor)}
```

Reply:
It also looks like this was applied to wrong getter (i.e., this should go on `getLocalMetaDataFileDescriptor` instead of `getMetaDataProvider`).

Reply:
my bad. fixed now.